### DISCUSSION TITLE
iOS 中的 block 是如何持有对象的 · /block-retain-object

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#279] [Announcements]
https://draveness.me/block-retain-object

**Comment by looseyi** [comment]
厉害了

**Comment by holysin** [comment]
嗨，灯塔，这好像有个错别字 "反而会标记 _string 属性"，请更正一下。

**Comment by draveness** [comment]
> @holysin
> 嗨，灯塔，这好像有个错别字 "反而会标记 _string 属性"，请更正一下。

我怎么没看出来..

**Comment by holysin** [comment]
> > @holysin
> > 嗨，灯塔，这好像有个错别字 "反而会标记 _string 属性"，请更正一下。
> 
> 我怎么没看出来..

应该是 _strong 吧
